maker/src/routes.rs

```diff
@@ -1,52 +1,70 @@
+use crate::bitmex::Quote;
+use anyhow::Result;
 use http_api_problem::HttpApiProblem;
+use http_api_problem::StatusCode;
 use rocket::serde::json::Json;
 use rocket::serde::Deserialize;
 use rocket::serde::Serialize;
-
-#[derive(Serialize, Deserialize, Debug)]
-pub struct Price(u64);
-
-impl From<u64> for Price {
-    fn from(val: u64) -> Self {
-        Self(val)
-    }
-}
+use rocket::State;
+use rust_decimal::prelude::ToPrimitive;
+use rust_decimal::Decimal;
+use rust_decimal_macros::dec;
+use tokio::sync::watch;
 
 #[derive(Serialize, Deserialize, Debug)]
 pub struct Offer {
-    bid: Price,
-    ask: Price,
+    bid: u64,
+    ask: u64,
 }
 
 #[derive(Serialize, Deserialize, Debug)]
 pub struct Offers {
     long: Offer,
     short: Offer,
-    index_price: Price,
+    index_price: u64,
 }
 
 #[rocket::get("/offers")]
-pub async fn get_offers() -> Result<Json<Offers>, HttpApiProblem> {
-    // TODO: Use real values instead of hard-coding
+pub async fn get_offers(
+    rx_quote_receiver: &State<watch::Receiver<Option<Quote>>>,
+) -> Result<Json<Offers>, HttpApiProblem> {
+    let rx_quote_receiver = rx_quote_receiver.inner().clone();
+    let quote = *rx_quote_receiver.borrow();
 
-    let long = Offer {
-        bid: 19000.into(),
-        ask: 21000.into(),
-    };
+    // TODO: take spread from clap
+    let spread = dec!(0.015);
```

Comment:
Given we don't run this in production we can remove or reduce it to a minimum :)